api/controllers/ExampleController.js

```diff
@@ -31,10 +31,10 @@ module.exports = {
       topic_id: topicId
     }).then(function(instance) {
       RedisClient.del('view_topic_' + topicId);
-      if(req.wantsJSON) {
+      if(req.wantsJSON && !(req.param('viewer_pane')==1)) {
```

Comment:
Yes, that's what I did at first, but I noticed it had some problems then with triple redirects somehow. There was a package 'Haven' that I tried that had a triple redirect and didn't want to load when I added the header. This was less clean but solved this issue. (the triple redifrect was because of an inconsitency in the elasticsearch database and the offline database though, so should not happen that often)